example/satosa/integration_test/settings.py

```diff
@@ -20,7 +20,8 @@
             "module": "pyeudiw.storage.mongo_storage",
             "class": "MongoStorage",
             "init_params": {
-                "url": "mongodb://localhost:27017/",
+                # according to Satosa-Saml2Spid demo
+                "url": "mongodb://satosa:thatpassword@localhost:27017/?timeoutMS=2000",
```

Comment:
there is a lot of duplication of the mongo settings in the tests

I need you @LadyCodesItBetter to merge them and import them instead of duplicating them